vendor/vendor.json

```diff
@@ -2329,35 +2329,19 @@
 			"version": "release-branch.go1.10",
 			"versionExact": "release-branch.go1.10"
 		},
-		{
-			"checksumSHA1": "dC7MpahR/U0T8CRcTypgrwO1Lg0=",
-			"path": "golang.org/x/tools/internal/fastwalk",
-			"revision": "04b5d21e00f1f47bd824a6ade581e7189bacde87",
-			"revisionTime": "2018-11-21T22:04:10Z",
-			"version": "release-branch.go1.10",
-			"versionExact": "release-branch.go1.10"
-		},
-		{
-			"checksumSHA1": "J+dCOMCPxRcqDPJtA3c1om9K8Q8=",
-			"path": "golang.org/x/tools/internal/gopathwalk",
-			"revision": "04b5d21e00f1f47bd824a6ade581e7189bacde87",
-			"revisionTime": "2018-11-21T22:04:10Z",
-			"version": "release-branch.go1.10",
-			"versionExact": "release-branch.go1.10"
-		},
```

Comment:
Out of curiosity - do you know why those were removed now?

Reply:
That's on me - they should have not made it in with the backport in https://github.com/elastic/apm-server/pull/1587